index.html

```diff
@@ -27,7 +27,7 @@
       </div>
       <div class="navbar">
         <nav class="grid-container">
-          <div class="title-bar align-justify hide-for-medium" data-responsive-toggle="navigation" data-hide-for="medium">
```

Comment:
I really don't want this to disappear. Is do you know how to accept or decline my changes?